internal/language/language.go

```diff
@@ -129,7 +129,7 @@ func MakeByNameAndVersion(name, version string) (Language, error) {
 	if strings.ToLower(name) == Python2.Requirement() {
 		parts := strings.Split(version, ".")
 		if len(parts) == 0 || parts[0] == "" {
-			return Unknown, locale.NewError("err_invalid_version", "Invalid langauage version number: {{.V0}}", version)
+			return Unknown, locale.NewError("err_invalid_language_version", "Invalid language version number: {{.V0}}", version)
```

Comment:
This is just an error that I found randomly while trying to activate a project with an invalid language version.